### PR TITLE
Make sure Deckard Cain can detect API Blueprint with a trailing whitespace

### PR DESCRIPTION
**Pull request by Baggz**

### CODE REVIEW

File: src/deckardcain.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const API_BLUEPRINT_HEADER = /^[\uFEFF]?(((VERSION:( |\t)2)|(FORMAT:( |\t)(X-)?1A))([\n\r]{1,2}|$))/i;
+const API_BLUEPRINT_HEADER = /^[\uFEFF]?(((VERSION:( |\t)2)|(FORMAT:( |\t)(X-)?1A))\s?([\n\r]{1,2}|$))/i;
```
Comment (honzajavorek) [inline src/deckardcain.js]: Wouldn't `\s*` be more bulletproof?

Comment (Baggz) [inline src/deckardcain.js]: You are right, @honzajavorek. Let me fix that...

Comment (Baggz) [inline src/deckardcain.js]: Fixed, sir.

**Comment by XVincentX** [comment]
[False save, USER DATA WASTED!](https://trello.com/c/sNq2oNJQ/258-false-save-user-data-wasted)

File: test/identify-test.js [hunk 2]
```diff
@@ -19,6 +19,50 @@ describe('API Blueprint', () => {
     });
   });
 
+  describe('with FORMAT header and a trailing whitespace', () => {
+    const source = dedent`
+      FORMAT: 1A
```
Comment (XVincentX) [inline test/identify-test.js]: I do not see the whitespace here...am I missing something?

**Comment by XVincentX** [comment]
Also, please bump the version to `0.3.3`

**Comment by honzajavorek** [comment]
When I was looking at the PR, the white space could be selected by cursor,
but otherwise was invisible.

**Pull request merged**